src/test/java/seedu/apollo/UiTest.java

```diff
@@ -15,7 +16,9 @@ class UiTest {
     void printList_normalInput_noExceptionThrown() throws IOException {
         Storage storage = new Storage("test.txt", "moduleData.txt");
         Ui ui = new Ui();
-        TaskList taskList = storage.loadTaskList(ui);
+        TaskList taskList = new TaskList();
+        ModuleList moduleList = new ModuleList();
+        TaskList taskList = storage.loadTaskList(ui, taskList, moduleList);
         assertDoesNotThrow(() -> ui.printList(taskList));
     }
 
```

Comment:
Possible pseudo algo:

1. Create new event.
2. Get start date/time and end date/time of event.
3. Between these two dates, iterate through every date and obtain a list of all events on that day.
4. Check all events in the list for each day if clashing using start and end time. If clashing return true and print ui message.
5. If no clashing, return false. No need to print.